src/index.js

```diff
@@ -46,7 +58,7 @@ export default {
   //		resize: function($element, layout) {
   //			console.log('resize>',$element,layout,$element.scope());
   //		},
-  template: '<bars-plus qv-extension />',
+  template: '<bars-plus class="qv-object-barsPlus" qv-extension />',
```

Comment:
Skip this part, css stuff is fixed by updating selector rather than adding another class in #4
```suggestion
  template: '<bars-plus qv-extension />',
```